resources/metadata/whitesource.yaml

```diff
@@ -407,6 +407,14 @@ spec:
           - name: custom/whitesourceProjectNames
             type: "[]string"
   containers:
+    - image: devxci/mbtci:1.0.14
+      workingDir: /project
+      env: [ ]
+      conditions:
+        - conditionRef: strings-equal
+          params:
+            - name: scanType
+              value: mta
```

Comment:
Will not work if only "buildTool" is specified. Also, not sure if it will if "buildTool" is taken from the CPE if unspecified.

Reply:
but this is also the same for the other docker images

Reply:
I check that it will work, because of: https://github.com/SAP/jenkins-library/blob/master/vars/whitesourceExecuteScan.groovy#L268

The scanType is resolved in groovy and passed to go as a parameter...

Reply:
I would suspect that it doesn't work there either, for the cases where the condition parameter is not mandatory and can be derived from another parameter.

Reply:
Yes, it will work! For "regular" wrappers, I think I would be right, though. For example Fortify.